tests/unit/amo/components/TestRatingManager.js

```diff
@@ -190,144 +192,163 @@ describe(__filename, () => {
         id: 2,
       },
     };
+
     const addon = createInternalAddon({
       ...fakeAddon,
       id: newReview.addon.id,
     });
 
-    const root = render({
-      apiState: { ...signedInApiState, token: 'new-token' },
-      version: { id: oldVersionId },
-      userId: 92345,
-      userReview: createInternalReview(newReview),
-      submitReview,
+    const store = createStoreWithLatestReview({
       addon,
-    });
-    return root.onSelectRating(newReview.rating).then(() => {
-      sinon.assert.called(submitReview);
-
-      // Make sure the review is submitted in a way where it will be
-      // newly created against the current version.
-      const call = submitReview.firstCall.args[0];
-      expect(call.reviewId).toEqual(undefined);
-      expect(call.versionId).toEqual(addon.current_version.id);
-      expect(call.rating).toEqual(newReview.rating);
-      expect(call.addonId).toEqual(newReview.addon.id);
-    });
+      review: newReview,
+      userId,
+      versionId: oldVersionId,
+    });
+
+    const root = render({ submitReview, addon, store });
+
+    return root
+      .instance()
+      .onSelectRating(newReview.rating)
+      .then(() => {
+        // Make sure the review is submitted in a way where it will be newly
+        // created against the current version.
+        sinon.assert.calledWith(
+          submitReview,
+          sinon.match({
+            reviewId: undefined,
+            versionId: addon.current_version.id,
+            rating: newReview.rating,
+            addonId: newReview.addon.id,
+          }),
+        );
+      });
   });
 
   it('renders and configures AddonReview after submitting a rating', () => {
-    const userReview = createInternalReview(fakeReview);
-    const FakeAddonReview = sinon.spy(() => <div />);
-    const root = render({ AddonReview: FakeAddonReview, userReview });
+    const store = createStoreWithLatestReview();
+    const root = render({ store });
 
-    sinon.assert.notCalled(FakeAddonReview);
+    expect(root.find(AddonReview)).toHaveLength(0);
 
-    return root.onSelectRating(5).then(() => {
-      sinon.assert.called(FakeAddonReview);
+    return root
+      .instance()
+      .onSelectRating(5)
+      .then(() => {
+        root.update();
 
-      const props = FakeAddonReview.firstCall.args[0];
-      expect(props.review).toEqual(userReview);
+        expect(root.find(AddonReview)).toHaveLength(1);
 
-      // Now make sure the callback is configured.
-      expect(root.state.showTextEntry).toEqual(true);
-      // Trigger the callback just like AddonReview would
-      // after completion.
-      props.onReviewSubmitted();
-      expect(root.state.showTextEntry).toEqual(false);
-    });
+        // Now make sure the callback is configured.
```

Comment:
I think this comment bitrotted prior to your patch. Maybe just remove it? I have no idea what it means.